lang/python/numpy/patches/001-stub-fe-values-mips-soft-float.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/numpy/core/src/npymath/ieee754.c.src b/numpy/core/src/npymath/ieee754.c.src
+index 3f66b24..87eb81f 100644
+--- a/numpy/core/src/npymath/ieee754.c.src
++++ b/numpy/core/src/npymath/ieee754.c.src
+@@ -789,6 +789,15 @@ int npy_clear_floatstatus_barrier(char *param)
+ /* General GCC code, should work on most platforms */
+ #  include <fenv.h>
+ 
++/* stub these for MIPS */
++#ifdef __mips_soft_float
++#define FE_INEXACT	0
++#define FE_UNDERFLOW	0
++#define FE_OVERFLOW	0
++#define FE_DIVBYZERO	0
++#define FE_INVALID	0
```

Comment:
I think these would be available when compiling with glibc? Or is `__mips_soft_float` only defined by musl?

Reply:
What's the point of stubbing when !soft_float is set in depends

Reply:
Oops
Pushed wrong branch

Reply:
> I think these would be available when compiling with glibc? Or is `__mips_soft_float` only defined by musl?

no idea; but if would make sense to do (even under the __mips_soft_float block)
```
#ifndef FE_UNDERFLOW
#define FE_UNDERFLOW 0
#endif 
```

Reply:
Probably these are defined in c++11 

https://en.cppreference.com/w/cpp/numeric/fenv/FE_exceptions

Reply:
See https://sourceware.org/git/?p=glibc.git;a=tree;f=sysdeps/mips/nofpu;hb=HEAD

Reply:
ok; will remove the `#ifndef __mips_soft_float`  block